spec/dummy/config/database.yml

```diff
@@ -4,9 +4,9 @@ development: &default
   encoding: unicode
   database: pg_saurus_dummy_development
   pool: 5
-  username: postgres
-  password: secret
-  host: localhost
+  #username: postgres
+  #password: secret
+  #host: localhost
```

Comment:
Maybe this  file should renamed to `database.yml.example` and `spec/dummy/config/database.yml` should be put to `.gitignore`